src/utils.ts

```diff
@@ -113,3 +113,8 @@ export function extractCredentialsFromUrl(
 		return resultOfErr("invalid URL");
 	}
 }
+
+export function sanitizeTorrentName(name: string): string {
+	const sanitizedName = name.replace("/", "");
+	return sanitizedName;
+}
```

Comment:
I don't think filenames can actually contain slashes - where are these torrent names coming from and how is it possible they have slashes?

https://stackoverflow.com/questions/9847288/is-it-possible-to-use-in-a-filename

Reply:
It's not the filename - it's the 'name' in the torrent file, which is what cross-seed is using to create the filename for the .torrent file.

I didn't think it was possible either, but I had the guy send me a screenshot of the torrent file in a torrent file editor, and it did have that in the name, was a season pack, and the files were shown in the root of the torrent, not in a folder.

When he loaded into his client, it went to the folder layout in qBittorrent though, I just applied same logic to Deluge as well.

Reply:
could we put this as a method on Metafile? `getFileSystemSafeName()` or something like that?

Reply:
Moved it to Metafile